src/Stripe/Services/Invoices/StripeInvoiceService.cs

```diff
@@ -63,10 +64,11 @@ public virtual IEnumerable<StripeInvoice> List(StripeInvoiceListOptions listOpti
             return Mapper<StripeInvoice>.MapCollectionFromJson(response);
         }
 
-        public virtual StripeInvoice Create(string customerId)
+        public virtual StripeInvoice Create(string customerId, StripeInvoiceCreateOptions createOptions = null)
```

Comment:
Off the top of my head, I think i'd remove the customer id from StripeInvoiceCreateOptions. Without looking through the other services, I believe most of them follow the convention of passing some sort of id before an options object. I would have to look through to verify I'm remembering correctly.